flutter_keyboard_visibility_web/pubspec.yaml

```diff
@@ -17,6 +17,7 @@ flutter:
 
 dependencies:
   flutter_keyboard_visibility_platform_interface: ^2.0.0
+  web: ^0.5.0
```

Comment:
Hello! We can already update to **web: ^1.0.0**

Reply:
adding web: ">=0.5.0 <2.0.0"  to support all existing and new version to avoid conflicts with outer packages